lib/eco/inclexer/inclexer.py

```diff
@@ -515,8 +515,10 @@ def relex(self, node):
             try:
                 token = next_token()
                 lookaheads.append(token[2])
-                if token[3][0] is startnode:
-                    past_startnode = True
+                for temp in token[3]:
```

Comment:
Nah I changed it now. I think it makes sense to have it in their in case there ever is a worst case scenario. The check is so cheap there's no reason not to put it in.